lib/Pipfile

```diff
@@ -17,7 +17,7 @@ testfixtures = "*"
 twine = "*"
 wheel = "*"
 # Doc builder packages:
-Sphinx = "==1.8.5"
+Sphinx = "==3.0.3"
```

Comment:
Why not `>3.0.3`?

Reply:
One fewer thing to think about for now, don't need to track down any build issues when we know this exact version is working correctly. We can evaluating floating it later when we have less to do